typescript/sdk/src/providers/transactions/transformer/ethersV5/EV5InterchainAccountTxTransformer.ts

```diff
@@ -56,19 +56,19 @@ export class EV5InterchainAccountTxTransformer
       this.props.config,
     );
 
-    const transformedTxs: ethers.PopulatedTransaction[] = [];
-    for (const txChain of objKeys(txChainsToInnerCalls)) {
+    const transformedTxs: Promise<ethers.PopulatedTransaction>[] = [];
+    objMap(txChainsToInnerCalls, (destination, innerCalls) => {
       transformedTxs.push(
-        await interchainAccountApp.getCallRemote({
+        interchainAccountApp.getCallRemote({
           chain: this.props.chain,
-          destination: txChain,
-          innerCalls: txChainsToInnerCalls[txChain],
+          destination,
+          innerCalls,
           config: this.props.config,
           hookMetadata: this.props.hookMetadata,
         }),
       );
-    }
+    });
 
-    return transformedTxs;
+    return Promise.all(transformedTxs);
```

Comment:
Do you think we'll need concurrency limits here (see `concurrentMap`)? Could these lists get big?

Reply:
very neat– thanks for pointing me to this util. these lists will be quite large sometimes, so this is a must.
added ✅